Frends.Python.ExecuteScript/Frends.Python.ExecuteScript.Tests/TestData/prepScript.sh

```
@@ -0,0 +1,4 @@
+﻿echo "Preparing environment..."
```

Shellcheck (0.10.0)

[error] This file has a UTF-8 BOM. Remove it with: LC_CTYPE=C sed '1s/^...//' < yourscript .

(SC1082)